8.0/Dockerfile.rhel8

```diff
@@ -0,0 +1,80 @@
+FROM ubi8/s2i-base:1
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+EXPOSE 8080
+EXPOSE 8443
+
+# Description
+# This image provides an Apache 2.4 + PHP 7.4 environment for running PHP applications.
+# Exposed ports:
+# * 8080 - alternative port for http
+
+ENV PHP_VERSION=7.4 \
```

Comment:
Old version, likely something to be changed later in a separate PR.

Reply:
Addressed by https://github.com/sclorg/s2i-php-container/pull/347/commits/b2bfb6c3f2fe144f39784203075ac243a0216f72